### PR TITLE
issue#62 fixed submit filter button in modal for tablet screen

### LINKED ISSUE #62: [AF4] Can't apply filter in modal on tablet screen
A button that should apply filters in modal filters doesn't display on tablet screens with bsVersion=4
image (13)

### PR DESCRIPTION
**Pull request by aleevas**
That should fix issue that described in #62

### CODE REVIEW

**Review by duozersk — CHANGES_REQUESTED** [review]
We need to apply the similar fix to the "filters-header" classes.

File: openy_af4_vue_app/src/components/filters/Filters.vue [hunk 1]
```diff
@@ -236,6 +240,12 @@ export default {
         !this.isEqual(this.selectedLocations, this.initialLocations) ||
         !this.isEqual(this.selectedActivities, this.initialActivities)
       )
+    },
+    headerClasses() {
+      return this.bsVersion === 4 ? 'd-none d-lg-flex' : 'hidden-md hidden-lg'
```
Comment (duozersk) [inline openy_af4_vue_app/src/components/filters/Filters.vue]: It should be 'hidden-xs hidden-sm' for BS 3...

**Review by duozersk — CHANGES_REQUESTED** [review]
Please be careful and attentive to details.

File: openy_af4_vue_app/src/components/filters/Filters.vue (hunk 1 above)
Comment (aleevas) [reply openy_af4_vue_app/src/components/filters/Filters.vue]: @duozersk thanks for notice
fixed.
Please check it again

**Review by duozersk — APPROVED** [review]

**Pull request merged**